expertsystem/topology.py

```diff
@@ -316,7 +316,7 @@ def __init__(
         super().__init__(nodes, edges)
         self.node_props: Dict[int, dict] = {}
         self.edge_props: Dict[int, _EdgeType] = {}
-        self.graph_element_properties_comparator: Optional[Callable] = None
+        self.graph_node_properties_comparator: Optional[Callable] = None
```

Comment:
Does the caller always have to specify the comparator? Maybe better to provide a default one?

Reply:
2d633a37a070d3397fb0bcddf633f28acb7239cc